Add note about changing the default value of  properties.message.time… (#2569)
risingwavelabs/risingwave-docs: docs/guides/create-sink-kafka.md

|ssl.endpoint.identification.algorithm | properties.ssl.endpoint.identification.algorithm | str |

:::note
+ Set `properties.ssl.endpoint.identification.algorithm` to `none` to bypass the verification of CA certificates and resolve SSL handshake failure. This parameter can be set to either `https` or `none`. By default, it is `https`.

> + Starting with version 2.0, the default value for `properties.message.timeout.ms` has changed from 5 seconds to **5 minutes**, aligning with the default setting in the official Kafka library.

Author: ```suggestion
+ Starting with version 2.0, the default value for `properties.message.timeout.ms` has changed from 5 seconds to **5 minutes**, aligning with the default setting in the [official Kafka library](https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md).
```

> + Starting with version 2.0, the default value for `properties.message.timeout.ms` has changed. It's now **5 minutes** instead of 5 seconds, matching the default setting in the official Kafka library.

Author: The detailed description for this parameter is in [the repo file](https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md). Not sure if it should be changed too.
<img width="822" alt="image" src="https://github.com/user-attachments/assets/4a552517-b9a6-4aca-8bcd-9092331fbcf5">

Reviewer: Yes, please link `official Kafka library` to the repo file.

> + Starting with version 2.0, the default value for `properties.message.timeout.ms` has changed. It's now **5 minutes** instead of 5 seconds, matching the default setting in the official Kafka library.

Reviewer 2: ```suggestion
+ Starting with version 2.0, the default value for `properties.message.timeout.ms` has changed from 5 seconds to **5 minutes**, aligning with the default setting in the official Kafka library.
```
